src/app/rate_limiter.ts

```diff
@@ -0,0 +1,39 @@
+import { asyncScheduler, Observable, of, SchedulerLike } from 'rxjs'
+import { concatMap, delay, switchMap } from 'rxjs/operators'
+
+export default class RateLimiter {
```

Comment:
I like the energy to do it but what is wrong with the library (besides being old)?

Reply:
It has some peer dependencies that conflict with other dependencies and it looks like those dependencies require Node<14

Reply:
Looking at it's [package.json](https://github.com/insidewhy/rxjs-ratelimiter/blob/master/package.json), the only peer dependency it has is for `rxjx` which you were going to using `override` to force to `"^7.0.0"`.